src/walletSdk/KeyManager/index.ts

```diff
@@ -0,0 +1,458 @@
+import { Networks, Transaction, WebAuth } from "@stellar/stellar-sdk";
+
+import { freighterHandler } from "./handlers/freighter";
+
+import { albedoHandler } from "./handlers/albedo";
+import { ledgerHandler } from "./handlers/ledger";
+import { plaintextKeyHandler } from "./handlers/plaintextKey";
+// TODO - fix trezor errors
+// import { trezorHandler } from "./handlers/trezor";
```

Comment:
the trezor code is throwing some errors so I'm commenting out for a next PR

@piyalbasu for this error:
```
Cannot find module 'stellar-sdk' from 'node_modules/@trezor/connect-plugin-stellar/lib/index.js'

    Require stack:
      node_modules/@trezor/connect-plugin-stellar/lib/index.js
      src/walletSdk/KeyManager/handlers/trezor.ts
      src/walletSdk/KeyManager/index.ts
      src/index.ts
      test/keyManager.test.ts
```
Because trezor has a peer [dependency of stellar-sdk](https://github.com/trezor/trezor-suite/blob/62c48b2cf019528d31903dffe31fb8edf4f1a081/packages/connect-plugin-stellar/package.json#L27) , instead of `@stellar/stellar-sdk` 

That's something we need to ask them to change right? or do you know of a way to bypass that?

Reply:
Vibrant is only using `KeyType.plaintextKey` so we should be fine removing trezor unless Freighter is using it?

Reply:
cool, I'll leave it commented out for now

looks like freighter is only using plaintextKey too atm